data/src/main/java/com/linkedin/data/schema/SchemaToJsonEncoder.java

```diff
@@ -127,6 +128,14 @@ public String getCurrentNamespace()
     return _currentNamespace;
   }
 
+  /**
+   * Configure the encoder to always use fully qualified names when encoding named type references.
```

Comment:
can you add when this should be set

Reply:
This is for LinkedIn specific use case, we should avoid adding those information in the open source code base.

Reply:
Updated the javadoc to provide more details.